tcp.rflx

```diff
@@ -0,0 +1,124 @@
+package TCP is
+
+    -- RFC 793
+
+    type Source_Port is mod 2**16;
+    type Destination_Port is mod 2**16;
+    type Sequence_Number is mod 2**32;
+    type Acknowledgment_Number is mod 2**32;
+    type Data_Offset is range 5 .. 2**4 - 1 with Size => 4;
+
+    -- Reserved for future use. Must be zero.
+    type Reserved is range 0 .. 0 with Size => 3;
+
+    type Window is mod 2**16;
+    type Checksum is mod 2**16;
+    type Urgent_Pointer is mod 2**16;
+
+    -- Timestamp and Window_Scale options are described in RFC 7323
+    -- Selective Acknowledgment Options (SACK) and SACK_Permitted are described in RFC 2018
+    -- No_Operation, End_Of_Options_List are described in RFC 793
+    -- Maximum_Segment_Size is described in RFC 793 and RFC 879
+
+    type Kind is
+       (End_Of_Options_List    => 16#00#,
+        No_Operation           => 16#01#,
+        Maximum_Segment_Size   => 16#02#,
+        Window_Scale           => 16#03#,
+        SACK_Permitted         => 16#04#,
+        SACK                   => 16#05#,
+        Timestamp              => 16#08#)
+    with Size => 8;
+
+    type Length is range 2 .. 2**8 - 1 with Size => 8;
+    type TS_Length is range 10 .. 10 with Size => 8;
+    type TSval is mod 2**32;
+    type TSecr is mod 2**32;
+    type Shift_cnt is mod 2**8;
+    type MaxSegSize is mod 2**16;
+
+
+    type Option is
```

Comment:
There should be no two consecutive blank lines.

Reply:
Changed in dc9f40f3b2fde6d4029438a000055e9658205250.